### ISSUE TITLE
Replace thread.get_ident() with twisted.python.threadable.getThreadID() in twisted/internet/test/test_win32events.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8548]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8548|
|Type|enhancement|
|Created|2016-07-01 09:47:36Z|
|Branch|https://github.com/twisted/twisted/tree/8548-rodrigc-thread|

```
import thread
```
is deprecated in Python2.  Trying to **import thread** in Python3 will result in an ImportError.

The deprecation is documented here:
https://www.python.org/dev/peps/pep-3108/#thread-deprecation

and users are encouraged to not use this module.

The fix required is the same as this one:
https://github.com/twisted/twisted/commit/506ee225c4d2164586678a597e7ac340f2b6e872
 done for [ticket:8421]

<details><summary>Searchable metadata</summary>

```
trac-id__8548 8548
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8548_rodrigc_thread 8548-rodrigc-thread
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467366456131701 1467366456131701
changetime__1467620139007989 1467620139007989
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/284

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks good. Thanks!

I have triggered the windows coverage from the new slave

https://buildbot.twistedmatrix.com/builders/win-64-py2.7-coverage

The new slave is waiting for a review to enter production... so for now is in the unsupported part.

Ready to merge!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|GitHub set status to `closed`|
|-|-|

In changeset 502f3a1e2cc125d214b6b7df5b173aa91fc7f966
```
#!CommitTicketReference repository="" revision="502f3a1e2cc125d214b6b7df5b173aa91fc7f966"
Merge pull request #284 from twisted/8548-rodrigc-thread: Fix test_win32events.py on Python3

Author: rodrigc
Reviewer: adiroiban
Fixes: #8548
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|GitHub commented|
|-|-|

In changeset c1d3adf8fa60fe8caad977c2604827b5cbf4eb08
```
#!CommitTicketReference repository="" revision="c1d3adf8fa60fe8caad977c2604827b5cbf4eb08"
Merge pull request #248 from twisted/8524-rodrigc-file: Replace file() with open() in twisted/protocols

Replace file() with open() in twisted/protocols

Author: rodrigc, Lukasa
Reviewer: adiroiban
Fixes: #8548
```
